content/100-getting-started/01-quickstart.mdx

```diff
@@ -427,6 +427,33 @@ If you want to move forward with Prisma ORM using your own PostgreSQL, MySQL, Mo
 
 Check out the [`prisma-examples`](https://github.com/prisma/prisma-examples/) repository on GitHub to see how Prisma ORM can be used with your favorite library. The repo contains examples with Express, NestJS, GraphQL as well as fullstack examples with Next.js and Vue.js, and a lot more.
 
+### Build real-time apps with Prisma Pulse
+
+[Prisma Pulse](/pulse) enables you to stream change events from your database in a type-safe way. Here are some example projects that show what you can build with it:
+
+| Demo                                                                     | Description                                                              |
```

Comment:
Should we just link to the GitHub readme section to not have to maintain this list of examples here, or do you think it's better to be explicit, because the example descriptions give you an idea of Pulse also?

Reply:
> because the example descriptions give you an idea of Pulse also

Exactly, I think it's good that the examples are descriptive and give an idea of Pulse use cases.

I don't think that keeping the tables in sync is such a big issues (and also, there's not really a _risk_ when they get out of sync since nothing will "break").